mmpose/core/post_processing/matching.py

```diff
@@ -145,7 +142,7 @@ def _grouping(bboxes, pose_preds, num_joints, box_scores):
             x0, y0, s0 = person[k][0]
             if s0 < 0.05:
                 continue
-            for g_id, g in kp_group.items():
+            for g_id, _ in kp_group.items():
```

Comment:
actually the `kp_group[g_id]` in this loop can be written as `g`

Reply:
btw in the beginning of this function, 
`kp_groups = {k: {} for k in range(num_joints)}`